src/Services/LDAPService.php

```diff
@@ -355,7 +378,7 @@ public function getUsers($attributes = [])
         $results = [];
 
         foreach ($searchLocations as $searchLocation) {
-            $records = $this->gateway->getUsers($searchLocation, Ldap::SEARCH_SCOPE_SUB, $attributes);
+            $records = $this->gateway->getUsersWithIterator($searchLocation, $attributes);
```

Comment:
We could use `$this->getGateway()->...` here now

Reply:
Yep good idea, have made that change throughout all of LDAPService

Reply:
Will leave you to mark resolved when you're satisfied, sorry!

Reply:
Still valid